apps/pie-storybook/stories/testing/pie-radio-group.test.stories.ts

```diff
@@ -69,4 +70,38 @@ const KeyboardNavigationTemplate = () => html`
     <p><pie-button size="small-productive" variant="secondary" data-test-id="btn-4">Button 4</pie-button></p>
     `;
 
+const DynamicSlotsTemplate = () => {
+    let counter = 2;
+
+    // Function to dynamically add radio buttons to the radio group
+    const onAddButtonClick = () => {
+        const radioGroup = document.querySelector('pie-radio-group[data-test-id="radio-group-1"]');
+
+        if (radioGroup) {
+            const newRadio = document.createElement('pie-radio');
+            newRadio.setAttribute('value', counter.toString());
+            newRadio.textContent = `Option ${counter}`;
+
+            radioGroup.appendChild(newRadio);
+
+            counter++;
+        }
+    };
+
+    return html`
+            <pie-radio-group name="radio-group-1" data-test-id="radio-group-1">
+                <pie-radio data-test-id="radio-1" value="1">Option 1</pie-radio>
+            </pie-radio-group>
+
+            <p>
```

Comment:
More out of interest – why is this wrapped in a `<p>`?

Reply:
Because I am too lazy to style it properly but it was bugging me

Reply:
(this is not a public facing story)

Reply:
It's a vicious pathway towards putting form elements inside iconButton slots 😆